database/seeders/RolesSeeder.php

```diff
@@ -26,16 +26,22 @@
 namespace Database\Seeders;
 
 use Illuminate\Database\Seeder;
-use Silber\Bouncer\BouncerFacade as Bouncer;
+use Spatie\Permission\Models\Role;
 
 class RolesSeeder extends Seeder
 {
     public function run(): void
     {
-        // set abilities for default rules
-        Bouncer::allow('admin')->everything();
-        Bouncer::allow(Bouncer::role()->firstOrCreate(['name' => 'global-read'], ['title' => 'Global Read']))
-            ->to('viewAny', '*', []);
-        Bouncer::role()->firstOrCreate(['name' => 'user'], ['title' => 'User']);
+        // Reset cached roles and permissions
+//        app()[\Spatie\Permission\PermissionRegistrar::class]->forgetCachedPermissions();
+
+        // create permissions
+//        Permission::create(['name' => 'edit articles']);
```

Comment:
^

Reply:
Comments here are intended to show future permissions.  Perhaps not needed, but more of a TODO for myself.

I'm thinking of defining permissions in a json file and iterating over them in the seeder.  Totally undecided as of now though.